### PR TITLE
MM2 Source/Checkpoint connector update

### PR DESCRIPTION
**Pull request by tomasz-sadura**
Updated MM2 connector docs with:
- `offset.lag.max` advanced setting for MM2 Source connector
- limitations section for MM2 Checkpoint connector

### CODE REVIEW

**Review by bochenekmartin — APPROVED** [review]

File: modules/deploy/pages/deployment-option/cloud/managed-connectors/create-mmaker-source-connector.adoc [hunk 1]
```diff
@@ -162,6 +162,10 @@ on the *Create Connector* page:
 | `Auto offset reset`
 | `consumer.auto.offset.reset`
 | What to do when there is no initial offset in Kafka, or if the current offset does not exist any more on the server (for example, because that data has been deleted). 'earliest' - automatically reset the offset to the earliest offset. 'latest' - automatically reset the offset to the latest offset. 'none' - throw exception to the consumer if no previous offset is found for the consumer's group.
+
+| `Offset lag max`
+| `offset.lag.max`
+| How out-of-sync a remote partition can be before it is resynced. It has impact on the MirrorMaker2 Checkpoint connector - it is the maximum lag for syncing consumer groups. The default is 100.
```
Comment (Feediver1) [inline modules/deploy/pages/deployment-option/cloud/managed-connectors/create-mmaker-source-connector.adoc]: 100 what? ms? sec? Please specify.

Comment (Feediver1) [inline modules/deploy/pages/deployment-option/cloud/managed-connectors/create-mmaker-source-connector.adoc]:
```suggestion
| How out-of-sync a remote partition can be before it is resynced. This setting impacts the MirrorMaker2 Checkpoint connector as it is the maximum lag for syncing consumer groups. The default is 100.
```

File: modules/deploy/pages/deployment-option/cloud/managed-connectors/create-mmaker-checkpoint-connector.adoc [hunk 2]
```diff
@@ -11,6 +11,12 @@ from other Kafka clusters.
 * A service account with read-only access to the external cluster is available.
 * The Kafka cluster topics connector is running for the same source cluster, with a matching configuration.
 
+== Limitations
+
+The MirrorMaker2 Checkpoint connector migrates (recently) active consumer groups. It does not migrate
```
Comment (Feediver1) [inline modules/deploy/pages/deployment-option/cloud/managed-connectors/create-mmaker-checkpoint-connector.adoc]: Having trouble what you mean here with the "recently" parenthetical? The docs don't specify that something was recently added--we either support the documented functionality or do not.

I've tried to rewrite this blurb a few times, but realized that it is simply unclear and I am not sure what you are trying to say. Can you try again? Sorry--want this to be both clear and accurate.

File: modules/deploy/pages/deployment-option/cloud/managed-connectors/create-mmaker-source-connector.adoc (hunk 1 above)
Comment (tomasz-sadura) [reply modules/deploy/pages/deployment-option/cloud/managed-connectors/create-mmaker-source-connector.adoc]: records

File: modules/deploy/pages/deployment-option/cloud/managed-connectors/create-mmaker-checkpoint-connector.adoc (hunk 2 above)
Comment (tomasz-sadura) [reply modules/deploy/pages/deployment-option/cloud/managed-connectors/create-mmaker-checkpoint-connector.adoc]: The first sentence is to give a gist of what to expect, while the second sentence explain in details what it actually means. Maybe we can remove the first sentence altogether?

File: modules/deploy/pages/deployment-option/cloud/managed-connectors/create-mmaker-checkpoint-connector.adoc [hunk 3]
```diff
@@ -11,6 +11,11 @@ from other Kafka clusters.
 * A service account with read-only access to the external cluster is available.
 * The Kafka cluster topics connector is running for the same source cluster, with a matching configuration.
 
+== Limitations
+
+The MirrorMaker2 Checkpoint connector does not migrate consumer group offsets for the offsets lower than the highest
+offsets synced by the MirrorMaker2 Source connector by the time the MirrorMaker2 Checkpoint connector is started.
```
Comment (Feediver1) [inline modules/deploy/pages/deployment-option/cloud/managed-connectors/create-mmaker-checkpoint-connector.adoc]:
```suggestion
The MirrorMaker2 Checkpoint connector does not migrate consumer group offsets that are lower than the highest
offsets synced by the MirrorMaker2 Source connector by the time the MirrorMaker2 Checkpoint connector is started.
```

**Review by Feediver1 — APPROVED** [review]

**Pull request merged**